### PR TITLE
MWPW-137960 : Refactor Marquee block test cases, and other improvments

### PR DESCRIPTION
**Pull request by skumar09**
[MWPW-137960](https://jira.corp.adobe.com/browse/MWPW-137960): Refactor Marquee block test cases 

PR includes

- Refactoring of Marquee block test cases
- Introducing style folder to read `variables.css` and` block.css` files to verify CSS / attributes 
   - Note: This is a bit of experiment and exploration, currently including both files under the styles folder; we won't be having this folder and files when we move test cases to the corresponding repository 
- Introducing visual comparison step at section/block level.
   - Note: This is also a bit of experiment and exploration to evaluate execution time with visual comparision
- Introducing validation for console log errors 
```
-     await test.step('step-3: Verify and log if any console errors', async () => {
      consoleErrors.length > 0 && console.log('[Console error]:', consoleErrors);      
      expect.soft(consoleErrors.length).toBe(0);      
    });
```

### CODE REVIEW

File: libs/cssparser.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
+import css from 'css';
+
+export function getCSSObjects(variableCSSContent, blcokCSSContent) {
+  // parse variableCSS
+  const parsedVariablesCSS = css.parse(variableCSSContent);
+  const parsedBlockCSS = css.parse(blcokCSSContent);
```
Comment (amauch-adobe) [inline libs/cssparser.js:6]: misspelled blockCSSContent, it is currently blcokCSSContent.

File: libs/cssparser.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
+import css from 'css';
+
+export function getCSSObjects(variableCSSContent, blcokCSSContent) {
```
Comment (amauch-adobe) [inline libs/cssparser.js:3]: misspelled blockCSSContent, it is currently blcokCSSContent.

File: libs/webutil.js [hunk 3]
```diff
@@ -91,19 +91,49 @@ exports.WebUtil = class WebUtil {
...
       Object.entries(cssProps).map(async ([property, expectedValue]) => {
         try {
+          const prop = property.toLocaleLowerCase();
+          if (prop.includes('color') || prop.includes('background') || prop.includes('border-color') ) {
+            // Convert the expected color value to RGB if it's in hex format
+            if (expectedValue.startsWith('#')) {
+              expectedValue = await this.hexToRgb(expectedValue);
+            }
+          }
+          //console.log('property : ', property ,'::', expectedValue);
```
Comment (amauch-adobe) [inline libs/webutil.js:136]: was this a debug/testing log statement that is no longer needed or should it be uncommented?

**Review by amauch-adobe — CHANGES_REQUESTED** [review]
See my small comments.  My biggest question/comment is what is our plan for housing these images.  I know this was mostly for experimentation, but I don't think we want to load up the Nala repo with "golden/base" images for comparison tests.  The same is for result images that are taken.  The original concept for saving images for Nala was to have a separate repository to house images that we could just pull down for the comparison tests.  I had already created a repo for this named [nala-screenshots](https://github.com/adobecom/nala-screenshots).  This recommendation came from this community [discussion](https://github.com/orgs/adobecom/discussions/561).

**Comment by skumar09** [comment]
Closing this PR, (updates are made as part of other pr's )

**Pull request closed without merging**